tests/common.py

```diff
@@ -168,3 +168,14 @@ def assert_not_found(self, response) -> None:
             response_json["detail"]
             == "You do not have permission to perform this action."
         )
+
+    def get_student_points(self, student_id: int) -> int:
```

Comment:
Consider moving this method to `TestPointsPost` and renaming to `_get_student_points`.
I mean - this is an useful helper, but the usage is not really generic.

Reply:
Done